### DISCUSSION TITLE
post/2022-09-04-grpc-generic-call/

### DISCUSSION THREAD

**Discussion by giscus** [zcong1993/blog#81] [Announcements]
# post/2022-09-04-grpc-generic-call/

本文介绍 Grpc 泛化调用, 即不使用 pb 生成的 server/client stub 代码实现调用.

https://nblog.cong.moe/post/2022-09-04-grpc-generic-call/

**Comment by shanezhiu** [comment]
第一感受就是有些鸡肋
